devtools/aptos-cargo-cli/src/bin/cargo-x.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright © Aptos Foundation
```

Comment:
nit: technically we should be adding the full license, but I recognize the linter doesn't currently enforce this 😢 
```
// Copyright © Aptos Foundation
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Good point @JoshLind, we should discuss this, because the implementation details are totally different. I only borrow the idea of calling cargo commands with the appropriate --package flags and implement it very differently. I use the public determinator and guppy crates in crates.io to find the affected packages to pass to the --package flag, so that attribution is implicit. Given this, I wonder if we need to change the license. wdyt?

The original code was removed here: https://github.com/aptos-labs/aptos-core/pull/2042

Reply:
Hmm.. I'm so-so. If there's no copy/pasta and things are sufficiently different, I don't think we need to worry too much 😄 But, if you feel like there's some overlap, it probably makes sense to use the shared license?

Reply:
I added the shared license to couple of files that look like there is an overlap, just to be safe.